src/frontend/komponenter/Fagsak/journalposter/JournalpostListe.tsx

```diff
@@ -165,12 +166,25 @@ const JournalpostListe: React.FC<IProps> = ({ bruker }) => {
     }
 
     if (journalposterRessurs.status === RessursStatus.SUKSESS) {
-        const journalposterMedOverstyrtDato = journalposterRessurs.data?.map(journalpost => ({
-            ...journalpost,
-            datoMottatt:
-                journalpost.datoMottatt ||
-                hentDatoRegistrertSendt(journalpost.relevanteDatoer, journalpost.journalposttype),
-        }));
+        const journalposterMedOverstyrtDato = journalposterRessurs.data?.map(
+            tilgangsstyrtJournalpost => {
+                const { journalpost } = tilgangsstyrtJournalpost;
+                const journalpostMedRiktigDatoMottatt = {
+                    ...journalpost,
+                    datoMottatt:
+                        journalpost.datoMottatt ||
+                        hentDatoRegistrertSendt(
+                            journalpost.relevanteDatoer,
+                            journalpost.journalposttype
+                        ),
+                };
+
+                return {
```

Comment:
Lurer på det samme som i BA https://github.com/navikt/familie-ba-sak-frontend/pull/3348#discussion_r1784007449

Reply:
Har fikset opp i koden for å forhåpentligvis gjøre den ryddigere. Er enig at denne koden kunne misforstås. 🙌